src/CMakeLists.txt

```diff
@@ -44,6 +44,15 @@ if (STK_USE_CUDA)
     )
 endif()
 
+if(STK_ITK_BRIDGE)
+    set(STK_SRCS
```

Comment:
I think the CMake configuration is starting to look like a total mess. We should probably have a proper CMake subfolder for each STK "module".

Reply:
Yeah, structuring the code is a good idea. Regarding the namespaces I'm a bit unsure. Having too many different namespaces (both nested and parallel) makes things messy, both for using them and implementing things inside them. Like having `stk::Volume vol = stk::io::read_volume()` is a bit weird. I kinda regret the mess I've created with `cuda` and `gpu`, but I didn't really see an alternative there.